website2/src/components/homePage/faq/index.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import { Text } from "@/components/text";
+import styles from "./styles.module.css";
+import { useState } from "react";
+
+const FAQ_ITEMS = [
+  {
+    question: "Lorem ipsum dolor sit amet",
+    answer: "Lorem ipsum dolor sit amet",
+  },
+  {
+    question: "Lorem ipsum dolor sit amet",
+    answer: "Lorem ipsum dolor sit amet",
+  },
+  {
+    question: "Lorem ipsum dolor sit amet",
+    answer: "Lorem ipsum dolor sit amet",
+  },
+  {
+    question: "Lorem ipsum dolor sit amet",
+    answer: "Lorem ipsum dolor sit amet",
+  },
+  {
+    question: "Lorem ipsum dolor sit amet",
+    answer: "Lorem ipsum dolor sit amet",
+  },
```

Comment:
I naively copied over the existing FAQ JSX fragments into this suggestion, but I think it'll probably need some modification since the components don't exist. I also think it's completely valid to do this in a future PR if you prefer to just leave these `lorem ipsum`s for now!

```suggestion
  {
    question: <>How does garn differ from docker, vagrant, etc.?</>,
    answer: <>
      <p>
        <Garn /> uses nix instead of containers or VMs to provide
        dependencies. These dependencies will all be installed into the nix
        store (in <code>/nix/store</code>) and will be provided from there
        in the environments that you create with <Garn />.
      </p>
      <p>
        That means that <Garn /> scripts and checks and everything else you
        do when developing with <Garn /> runs natively on your machine. So
        there is no overhead that you would get when using VMs or containers
        on non-linux hosts.
      </p>
      <p>
        That also means that -- when entering an environment with garn enter
        -- you can use your personal editor or other globally installed
        tools with no additional work.
      </p>
    </>,
  },
  {
    question: <>How does garn differ from 'npm', 'cabal' and other language-specific tools?</>,
    answer: <>
      <Garn /> is language agnostic. That means you can use it in projects
      that mix multiple languages and have the same experience and
      workflows for all sub-projects. It also means that programmers that
      are unfamiliar with the toolchain of a project can use <Garn /> to
      get started quickly.
    </>,
  },
  {
    question: <>What languages does <Garn /> support?</>,
    answer: <>
      <Garn /> currently supports <strong>Go</strong>,{" "}
      <strong>Npm</strong> and <strong>Haskell</strong>. If you'd like to
      see support for other languages or toolchains please{" "}
      <a href="https://github.com/garnix-io/garn/issues">let us know</a>.
    </>,
  },
  {
    question: <>How can I configure LSP to get error messages and auto-completion for garn.ts in my editor?</>,
    answer: <>
      <p>
        <code>garn.ts</code> files are powered by{" "}
        <a href="https://deno.com/">Deno</a>. A lot of the convenience and
        power for editing your <code>garn.ts</code> files comes from having
        a working Deno LSP. There are two ways of setting up LSP for editing{" "}
        <code>garn.ts</code> files: using `garn edit`, which provides a
        properly-configured editor, or using <Garn /> to provide{" "}
        <code>deno</code> and setting up your editor configuration yourself.
      </p>
      <h3>
        <code>garn edit</code>
      </h3>
      <p>
        <Garn /> can download and configure{" "}
        <a href="https://vscodium.com/">vscodium</a> for you &mdash; just
        type <code>garn edit</code>. It will spin up a{" "}
        <a href="https://vscodium.com/">vscodium</a> editor that is
        pre-configured for editing <code>garn.ts</code> files. It won't use
        or modify your local vscodium settings, if you have any. You can can
        add it to your <code>garn.ts</code> file like this:
      </p>
      <h3>Installing the Deno LSP and configuring your editor</h3>
      <p>
        Alternatively you can install <code>deno</code> (including the Deno
        LSP) with <Garn /> itself:
      </p>
      <Code
        header="garn.ts"
        lineNumbers
        code={`import * as garn from "https://garn.io/ts/v0.0.19/mod.ts";

  export const deno = garn.mkProject({
  description: "garn configuration environment",
  defaultEnvironment: garn.emptyEnvironment.withDevTools([pkgs.deno]),
  }, {});`}
      />
      <p>
        <code>garn enter deno</code> will then drop you in a shell where{" "}
        <code>deno</code> is available.
      </p>
      <p>
        For configuring your editor to use Deno's LSP refer to{" "}
        <a href="https://docs.deno.com/runtime/manual/getting_started/setup_your_environment">
          Deno's environment setup documentation
        </a>.
      </p>
    </>,
  },
```

Reply:
yah. i'll need to do some other work to get some of the text formatting to come over and to look good, i'll hold off on this for now until I do the content pass at the end